elyra/pipeline/component_parser_airflow.py

```diff
@@ -142,13 +142,13 @@ def _parse_properties(self, component_definition: str, component_class: str) ->
 
             # Search for :param [param] in class doctring to get description
             description = ""
-            param_regex = re.compile(f":param {arg}:" + r"([\s\S]*?(?=:type|:param))")
+            param_regex = re.compile(f":param {arg}:" + r"([\s\S]*?(?=:type|:param|\"\"\"|'''|\.\.))")
             match = param_regex.search(class_content)
             if match:
                 description = match.group(1).strip().replace("\"", "'")
 
             # Amend description to include type information
-            description = self._get_description_with_type_hint(description, type)
+            description = self._get_description_with_type_hint(type, description)
```

Comment:
I wonder if this should be named `_format_description()` since its not really _getting_ anything, and, as was a natural instinct, have `description` first, then `type`.  I think using keyword arguments might help:
```python
def _format_description(description: str, type: str) -> str:
```
Since the kfp parser could call this with a `None` description (airflow defaults to `""`), I would update its call such that the dictionary `get()` method uses a default value of `""` to ensure its at least the empty string.